macros/edr/materializations/model/common.sql

```diff
@@ -0,0 +1,28 @@
+{% macro query_table_metrics() %}
+  {% set query %}
+    select count(*) as row_count
```

Comment:
The freshness metrics (model run timestamp) can be calculated from this metric.
Should we add an empty `freshness` or `build` metric for clarity?